x/emissions/module/rewards/rewards_internal.go

```diff
@@ -445,6 +445,13 @@ func GetAllReputersOutput(
 	coefficients := make([]alloraMath.Dec, len(initialCoefficients))
 	copy(coefficients, initialCoefficients)
 
+	// Check if there's only one reputer and the listening coefficient is zero
+	// If so, set the listening coefficient to 0.5
+	// This is to ensure there's no bugs in the scores calculation
+	if len(coefficients) == 1 && coefficients[0].IsZero() {
+		coefficients[0] = alloraMath.MustNewDecFromString("0.5")
```

Comment:
According to Diederik, the other alternative would be to find the value closest to zero from the previous epoch or 0.5. I chose the latter option because of the simplicity, the other option would be necessary to keep track of all the previous epoch active reputers.

Reply:
Oh yes makes sense